CMakeLists.txt

```diff
@@ -245,5 +244,7 @@ foreach(DLL_FILE ${onnxruntime_libs})
   )
 endforeach()
 
+include(cmake/package.cmake)
```

Comment:
This file currently is not used by Ci, but planning to use it in the future because cmake can provide consistency between Github and ADO.